### PR TITLE
Added travis file.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+language: android
+
+android:
+  components:
+    - build-tools-21.1.2
+    - extra-android-m2repository
+    - extra-google-m2repository
```
Comment (JakeWharton) [inline .travis.yml]: You don't need this. Probably not the android extras either.

**Comment by JakeWharton** [comment]
:shipit:

File: .travis.yml (hunk 1 above)
Comment (zach-klippenstein) [inline .travis.yml]: Removed both extras repos.

**Comment by zach-klippenstein** [comment]
Upgraded to Robolectric 2.3 to fix the java8 build.

**Pull request merged**